project/plugins.sbt

```diff
@@ -4,6 +4,11 @@ resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.15.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.2.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.20")
-addSbtPlugin("org.scoverage"     % "sbt-scoverage"         % "1.9.3")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.21")
+addSbtPlugin("org.scoverage"     % "sbt-scoverage"         % "2.0.9")
 addSbtPlugin("org.scalastyle"   %% "scalastyle-sbt-plugin" % "1.0.0")
+addSbtPlugin("ch.epfl.scala"     % "sbt-bloop"             % "1.5.11")
+addSbtPlugin("org.scalameta"     % "sbt-scalafmt"          % "2.5.2")
+addSbtPlugin("ch.epfl.scala"     % "sbt-scalafix"          % "0.11.1")
```

Comment:
You fell for the missing % - it should be %% ...

Reply:
Fixed